### ISSUE TITLE
I setup my worker with a once every five minute call from crontab but I am seeing a large backlog.

### ISSUE THREAD

**Issue by 4nanook** [friendica/friendica#9087]
### Expected behavior
The instructions said to schedule every 5-10 minutes, I scheduled every five minutes and expected this would suffice.
### Actual behavior
I have 625 items in the worker queue.  I changed the scheduling to every minute.  Queue has not gotten shorter.
### Steps to reproduce the problem
It happens by itself.
### Friendica version you encountered the problem
This is Friendica, version 2020.09-dev that is running at the web location https://friendica.eskimo.com. The database version is 1364, the post update version is 1341.
### Friendica source (git, zip)
git
### PHP version
PHP 7.4.9 (cli) (built: Aug  7 2020 14:30:01) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.9, Copyright (c), by Zend Technologies

### SQL version
mysql  Ver 15.1 Distrib 10.3.22-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2

**Comment by annando** [comment]
625 doesn't sound like any problem. The system does many background activities, like regularly checking remote contacts and other stuff. Also posts that couldn't be delivered the first time are retried several times. The question is more if your posts are delivered fast.

**Comment by 4nanook** [comment]
Now is up to 999 even though scheduled once per minute.

-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
  Eskimo North Linux Friendly Internet Access, Shell Accounts, and Hosting.
    Knowledgeable human assistance, not telephone trees or script readers.
  See our web site: http://www.eskimo.com/ (206) 812-0051 or (800) 246-6874.

**Comment by MrPetovan** [comment]
Mine is at 622 and my worker is running every minute if possible, so it doesn't sound like there's a problem.

**Comment by 4nanook** [comment]
Mine is running about two hours behind even though no resource is
saturated as near as I can tell.  Load of about eight on a 12 thread machine,
using about 8GB of 59GB of RAM, disk I/O relatively light, no slow queries
in database, etc.  Cron does not allow more than 1/minute so don't know how
to speed it up anymore.

-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
  Eskimo North Linux Friendly Internet Access, Shell Accounts, and Hosting.
    Knowledgeable human assistance, not telephone trees or script readers.
  See our web site: http://www.eskimo.com/ (206) 812-0051 or (800) 246-6874.

**Comment by annando** [comment]
A load of 8 is slightly high. How many parallel workers have you set up? And have you enabled the fast lane?

**Comment by 4nanook** [comment]
The load of 8 isn't high on a 12 thread machine, that still means 4
threads are completely unoccupied.  Anyway, initially I set it up as 12,
and it was running about two hours behind, jacked that up to 24 and it cleared
the queue and caught up.

-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
  Eskimo North Linux Friendly Internet Access, Shell Accounts, and Hosting.
    Knowledgeable human assistance, not telephone trees or script readers.
  See our web site: http://www.eskimo.com/ (206) 812-0051 or (800) 246-6874.

**Comment by annando** [comment]
My server squeet.me has got a load between 1 and 3 on a (virtual) machine with 6 cores, 32gb memory and 30 parallel workers. When the number of workers is set to 20, the load is between 1 and 2.

**Comment by 4nanook** [comment]
This machine is not dedicated to friendica, that is only one application
running on the box so other things contribute to the load.  Still not a 
problem, friendica loads in 600ms, my WordPress website loads in 255ms, there
are ZERO slow queries in MariaDB stats, all is good, just needed to jack the 
workers up from 12 to 24, now nothing is more than a minute stale, as opposed 
to two hours with 12 workers.

-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
  Eskimo North Linux Friendly Internet Access, Shell Accounts, and Hosting.
    Knowledgeable human assistance, not telephone trees or script readers.
  See our web site: http://www.eskimo.com/ (206) 812-0051 or (800) 246-6874.

**Comment by tobiasd** [comment]
Do you have activated the Fastlane for the Worker in the admin settings?

**Comment by 4nanook** [comment]
Yes but still work was running approximately 2 hours behind.  But with 24 workers it is staying current within one minute, so all good.

**Comment by 4nanook** [comment]
Issue seems to have been an overloaded database even though I was not seeing that many slow queries.  Upgraded to a flash drive that supports atomic writes and problem went away.

**Comment by 4nanook** [comment]
Since it seems to have been a hardware issue on my end, I am closing this report.
